docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2176 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    ibis.options.repr.interactive.max_columns = None
</pre><p>In 5.0 this won't be necessary I believe. We set the limit to something arbitrarily large (1,000,000 IIRC) when inside a Jupyter notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>But for this blog post, I'll just remove this to keep it simple.</p>

Reply:
SGTM!